build.gradle

```diff
@@ -44,6 +44,7 @@ dependencies {
     compile 'ch.qos.logback:logback-classic:1.3.+'
     compile 'ch.qos.logback:logback-core:1.3.+'
     testCompile group: 'org.mockito', name: 'mockito-all', version: '1.9.+'
+    testCompile group: 'com.nhaarman', name: 'mockito-kotlin', version: '1.6.+'
```

Comment:
There's [mockito-kotlin2](https://github.com/nhaarman/mockito-kotlin) (which I guess uses mockito 2). Any sense to switch (and thus to mockito 2 as well) ?

Reply:
Updated lib.